.github/workflows/build-check-install.yaml

```
@@ -146,28 +146,28 @@ on:
         default: ""
       skip-desc-branch:
         description: |
-          Passed to `insightsengineering/actions/setup-r-dependencies`.
+          Passed to `insightsengineering/setup-r-dependencies/blob/main/action.yaml`.
```

I think I would stick to the org/repo
```suggestion
          Passed to `insightsengineering/setup-r-dependencies`.
```
GitHub reference is specified elsewhere (down below in the `use:` statement) so not to duplicate